Public/Src/Engine/Scheduler/Graph/PipGraph.Serialization.cs

```diff
@@ -26,7 +26,7 @@ private sealed class SerializedState
             public readonly ConcurrentBigMap<ModuleId, NodeId> Modules;
             public readonly ConcurrentBigMap<FileArtifact, NodeId> PipProducers;
             public readonly ConcurrentBigMap<DirectoryArtifact, NodeId> OpaqueDirectoryProducers;
-            public readonly ConcurrentBigSet<AbsolutePath> OutputDirectoryRoots;
+            public readonly ConcurrentBigMap<AbsolutePath, bool> OutputDirectoryRoots;

```

Comment:
>bool [](start = 59, length = 4)

nit: maybe a comment here saying what this boolean is about?

Reply:
agreed